### PR TITLE
Build testapps workflow

### PR DESCRIPTION
**Pull request by sunmou99**
Set up workflow to build Unity Testapps. 
Workflow passed with Unity 2017.4 and all platforms: https://github.com/firebase/firebase-unity-sdk/actions/runs/1121879986
Test with latest Unity SDK: https://firebase.google.com/docs/unity/setup
Now the script can build iOS .app (for simulator) & .ipa (for real device) in sequence.
To save the Unity license, `input.platforms` is not added to build matrix.

TODO: Better usage of Unity license. And speed up the overall build time.

### CODE REVIEW

**Comment by sunmou99** [comment]
> We found a Contributor License Agreement for you (the sender of this pull request), but were unable to find agreements for all the commit author(s) or Co-authors. If you authored these, maybe you used a different email address in the git commits than was used to sign the CLA ([login here](https://cla.developers.google.com/) to double check)? If these were authored by someone else, then they will need to sign a CLA as well, and confirm that they're okay with these being contributed to Google.
> In order to pass this check, please resolve this problem and then comment `@googlebot I fixed it.`. If the bot doesn't comment, it means it doesn't think anything has changed.
> 
> ℹ️ **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Ffirebase%2Ffirebase-unity-sdk%2Fpull%2F46) for more info**.

@googlebot

**Comment by sunmou99** [comment]
@googlebot I fixed it.

File: .github/workflows/integration_tests.yml [hunk 1]
```diff
@@ -77,7 +87,27 @@ jobs:
...
-          cat testapps/activate_license.log
+      - name: Prepare for integration tests
+        timeout-minutes: 10
+        shell: bash
+        run: |
+          python scripts/gha/restore_secrets.py --passphrase "${{ secrets.TEST_SECRET }}"
+      - name: Build integration tests
+        env:
+          UNITY_ANDROID_SDK: /Users/runner/Library/Android/sdk
+          # UNITY_ANDROID_NDK: ${{ steps.setup-ndk.outputs.ndk-path }}
```
Comment (yifanyang) [inline .github/workflows/integration_tests.yml:98]: Which one should be used?

File: scripts/gha/build_testapps.py [hunk 2]
```diff
@@ -618,8 +636,8 @@ def _add_menu_scene(dir_helper):
 
 
 def _add_automated_test_runner(dir_helper):
-  """Copies automated_testapp."""
-  logging.info("Adding automated_testapp to sample...")
+  """Copies automated_testpp."""
+  logging.info("Adding automated_testpp to sample...")
```
Comment (yifanyang) [inline scripts/gha/build_testapps.py:640]:
```suggestion
  """Copies automated_testapp."""
  logging.info("Adding automated_testapp to sample...")
```

File: .github/workflows/integration_tests.yml (hunk 1 above)
Comment (sunmou99) [reply .github/workflows/integration_tests.yml]: This is for test purposes. Removed

File: .github/workflows/integration_tests.yml [hunk 3]
```diff
@@ -77,7 +87,24 @@ jobs:
...
+      - name: Prepare for integration tests
+        timeout-minutes: 10
+        shell: bash
+        run: |
+          python scripts/gha/restore_secrets.py --passphrase "${{ secrets.TEST_SECRET }}"
+      - name: Build integration tests
+        shell: bash
+        run: |
+          # TODO: Download unity SDK first
+          curl https://dl.google.com/firebase/sdk/unity/firebase_unity_sdk_8.1.0.zip -o ~/Downloads/firebase_unity_sdk.zip
```
Comment (a-maurice) [inline .github/workflows/integration_tests.yml:99]: Can you change the download url to a workflow parameter?

**Review by a-maurice — APPROVED** [review]

**Pull request merged**